tests/rados/test_replica1.py

```diff
@@ -0,0 +1,161 @@
+""" Test module to verify replica-1 non-resilient pool functionalities"""
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.crushtool_workflows import CrushToolWorkflows
+from tests.rados.monitor_configurations import MonConfigMethods
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+from utility.utils import generate_unique_id
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    # CEPH-83575297
+
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    crush_obj = CrushToolWorkflows(node=cephadm)
+    mon_obj = MonConfigMethods(rados_obj=rados_obj)
+    rbd_obj = Rbd(**kw)
+    repli_pools = []
+    zone_unique = generate_unique_id(length=4)
+
+    log.info("Running test case to verify replica-1 non-resilient pool")
+
+    try:
+        # Fetch all the OSDs on the cluster
+        out, _ = cephadm.shell(args=["ceph osd ls"])
+        osd_list = out.strip().split("\n")
+        log.debug(f"List of OSDs: {osd_list}")
+
+        # remove the device class for all the OSDs
+        device_rm_cmd = "ceph osd crush rm-device-class " + " ".join(osd_list)
```

Comment:
Let's create a usable method in ` crushtool_workflows.py ` for removal of crush device class.